src/python/WMQuality/Emulators/SiteDBClient/MockSiteDBApi.py

```diff
@@ -0,0 +1,54 @@
+#! /usr/bin/env python
+
+"""
+Version of SiteDB.SiteDBJSON intended to be used with mock or unittest.mock
+"""
+from __future__ import (division, print_function)
+
+from WMCore.WMBase import getTestBase
+from RestClient.ErrorHandling.RestClientExceptions import HTTPError
+import os
+import json
+
+# Read in the data just once so that we don't have to do it for every test (in __init__)
+mockData = {}
+globalFile = os.path.join(getTestBase(), '..', 'data', 'Mock', 'SiteDBMockData.json')
+print(globalFile)
```

Comment:
Definitely don't want it printing a filename every time. You could (should) add logging.info here instead